frontend/server/src/Controllers/Contest.php

```diff
@@ -1379,6 +1379,8 @@ public static function getContestNewForTypeScript(
                     'hasVisitedSection' => \OmegaUp\UITools::hasVisitedSection(
                         'has-visited-create-contest'
                     ),
+                    'isSystemAdminOrSupport' => \OmegaUp\Authorization::isSystemAdmin($r->identity) || 
+                        \OmegaUp\Authorization::isSupportTeamMember($r->identity),
```

Comment:
Since the authorization system we use is inherited, could you omit the call to the isSystemAdmin function? This way, you would still grant access to the sysadmin.

Additionally, could you rename it to `canSetRecommended`?

In the end, it should look like this:
```suggestion
                    'isSystemAdminOrSupport' => \OmegaUp\Authorization::isSupportTeamMember($r->identity),
```

You can apply this in the other places where you use it.